packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -92,6 +92,9 @@ export function ListToVirtualizedList(props) {
 							columnData: column.data,
 						});
 					}
+					if (column.disableSort) {
```

Comment:
You can remove the condition altogether and assign column.disableSort to  cProps.disableSort